drizzle-orm/src/mysql-core/dialect.ts

```diff
@@ -26,6 +26,14 @@ import type { MySqlSelectConfig, MySqlSelectJoinConfig, SelectedFieldsOrdered }
 import type { MySqlUpdateConfig } from './query-builders/update.ts';
 import type { MySqlSession } from './session.ts';
 import { MySqlTable } from './table.ts';
+
+// TODO find out how to use all/values. Seems like I need those functions
+// Build project
+// copy runtime tests to be sure it's working
+
+// Add mysql to drizzle-kit
+
+// Add Planetscale Driver and create example repo
```

Comment:
This might be a left over comment from before last release. Or did you add it?

Reply:
The former is the case. Will remove.